src/contribute/contribution-guide.md

```diff
@@ -8,14 +8,31 @@ redirect_from: /contribution-guide/
 
 # Apache Beam Contribution Guide
 
-The Apache Beam community welcomes contributions from anyone with a passion for
-data processing! Beam has many different opportunities for contributions --
-write new examples, add new user-facing libraries (new statistical libraries,
-new IO connectors, etc), work on the core programming model, build specific
-runners (Apache Apex, Apache Flink, Apache Spark, Google Cloud Dataflow, etc),
-or participate on the documentation effort.
-
-We use a review-then-commit workflow in Beam for all contributions.
+The Apache Beam community welcomes contributions from anyone!
+
+There are lots of opportunities:
+
+ - write new examples
+ - improve the documentation
+ - add new user-facing libraries (new statistical libraries, new IO connectors,
+   etc)
+ - improve your favorite language SDK (Java, Python, Go, etc)
+ - improve specific runners (Apache Apex, Apache Flink, Apache Spark, Google
+   Cloud Dataflow, etc)
+ - work on the core programming model (what is a Beam pipeline and how does it
+   run?)
+
+Most importantly, if you have an idea of how to contribute, then do it! 
+
+For a list of open starter tasks, check
+[https://s.apache.org/beam-starter-tasks](https://s.apache.org/beam-starter-tasks).
```

Comment:
This link doesn't seem to be working for me.

Reply:
Yea I don't really quite understand what I have messed up in the markdown.

Reply:
The markdown looks fine to me, but I'm getting "The requested filter doesn't exist or is private. Try logging in." when going to the link.

Reply:
Ah, the same UI fail I've had every time - select who to share it with, click "Save". Forgot to click "+Add" first.

Reply:
Try now?